src/irmin-io/ir_io.mli

```diff
@@ -0,0 +1,7 @@
+module Sync: Git.Sync.IO
```

Comment:
The apache license says that you should only add a license file at the root of the repo. I'm happy to add license headers in the file too if that's necessary.